### PR TITLE
Replaced createSlot with createSlots returning HTMLElement_s_

### PR DESCRIPTION
**Pull request by jeteve**
## What does this change?

This implements https://trello.com/c/M2rgtH0O/158-amend-location-of-blockthrough-placement-to-existing-ad-slots

### CODE REVIEW

File: common/app/views/fragments/commercial/adSlot.scala.html [hunk 1]
```diff
@@ -25,9 +25,9 @@
...
-        @Commercial.getBlockthroughElementUid(name).map { uid =>
-            <span class="bt-uid-tg" uid="@uid" style="display: none !important"></span>
-        }
-    }
 </div>
+@if(!request.isAmp) {
+    @Commercial.getBlockthroughElementUid(name).map { uid =>
+<span class="bt-uid-tg" uid="@uid" style="display: none !important"></span>
+    }
+}
```
Comment (kelvin-chappell) [inline common/app/views/fragments/commercial/adSlot.scala.html]: Is this a deliberate change?

File: static/src/javascripts/projects/commercial/modules/article-body-adverts.js [hunk 2]
```diff
@@ -33,17 +33,24 @@ const insertAdAtPara = (
...
+                if (para.parentNode) {
+                    para.parentNode.insertBefore(ad, para);
+                }
+            })
+        )
         .then(() => {
             const shouldForceDisplay = ['im', 'carrot'].includes(name);
-            addSlot(ad, shouldForceDisplay);
+            // Only add the first ad (the DFP one) to GTP
+            addSlot(ads[0], shouldForceDisplay);
```
Comment (kelvin-chappell) [inline static/src/javascripts/projects/commercial/modules/article-body-adverts.js]: What are the other ads?

**Review by kelvin-chappell — APPROVED** [review]
Looks like one file is changed unnecessarily but otherwise 👍

File: common/app/views/fragments/commercial/adSlot.scala.html (hunk 1 above)
Comment (jeteve) [reply common/app/views/fragments/commercial/adSlot.scala.html]: Yes, the 'Blockthrough' span must be outside the main adslot for blockthrought to work. See https://trello.com/c/M2rgtH0O/158-amend-placement-of-blockthrough-placement-to-existing-ad-slots

Comment (kelvin-chappell) [reply common/app/views/fragments/commercial/adSlot.scala.html]: I see.  So it's moved outside the div.  But indenting looks a bit strange.

Comment (jeteve) [reply common/app/views/fragments/commercial/adSlot.scala.html]: I wanted it to be at the same indentation level as the above div, so it looks neat in the page source.

Comment (kelvin-chappell) [reply common/app/views/fragments/commercial/adSlot.scala.html]: OK 👍

**Pull request merged**